docroot/modules/custom/mass_redirects/src/Plugin/QueueWorker/MassRedirectsRepoint.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace Drupal\mass_redirects\Plugin\QueueWorker;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItem;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Drupal\Core\Url;
+use Drupal\mass_content\Field\FieldType\DynamicLinkItem;
+use Drupal\text\Plugin\Field\FieldType\TextLongItem;
+use Drupal\text\Plugin\Field\FieldType\TextWithSummaryItem;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Database\Connection;
+
+/**
+ * Processes mass_redirects_repoint queue.
+ *
+ * @QueueWorker(
+ *   id = "mass_redirects_repoint",
+ *   title = @Translation("Mass redirects repoint"),
+ *   cron = {"time" = 300}
```

Comment:
Lets remove this line, and use a dedicated scheduled job for this queue. We cant rely on the main cron job to do so much, and thats fixed on a 5 min period.

Reply:
Done